### PR TITLE
Add modal contact-us form for /openstack/storage

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

Add modal form for /openstack/storage

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:http://0.0.0.0:8001/openstack/storage#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](chrome-extension://klbibkeccnjlkjkiokjodocebajanakg/suspended.html#ttl=storage%20-%20Google%20Docs&pos=0&uri=https://docs.google.com/document/d/1Pfay_PfsE7Tw8gU82wENRCcCUEXbe2gf804uGzM_PqY/edit#) and test the output

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1409

### CODE REVIEW

File: templates/shared/forms/interactive/_storage.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            <input type="checkbox" id="storage-solutions-Rook.io">
+            <label for="storage-solutions-Rook.io">Rook.io</label>
+          </div>
+          <div class="col-3 u-sv3">
+            <input type="checkbox" id="storage-solutions-Other-NAS">
+            <label for="storage-solutions-Other-NAS">Other NAS</label>
+            <input type="checkbox" id="storage-solutions-Other-SAN">
+            <label for="storage-solutions-Other-SAN">Other SAN</label>
+            <input type="checkbox" id="storage-solutions-Local-storage">
+            <label for="storage-solutions-Local storage">Local-storage</label>
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/_storage.html]: The for should be: "storage-solutions-Local-storage"

File: templates/shared/forms/interactive/_storage.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+            <label for="compute-infrastructure-Kubernetes">Kubernetes</label>
+            <input type="checkbox" id="compute-infrastructure-KVM ">
+            <label for="compute-infrastructure-KVM ">KVM </label>
+          </div>
+          <div class="col-3 u-sv3">
+            <input type="checkbox" id="compute-infrastructure-LXD">
+            <label for="compute-infrastructure-LXD">LXD</label>
+            <input type="checkbox" id="compute-infrastructure-Docker">
+            <label for="compute-infrastructure-Docker">Docker</label>
+            <input type="checkbox" id="compute-infrastructure-Public cloud">
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/_storage.html]: Space in id, should be: compute-infrastructure-Public-cloud

File: templates/shared/forms/interactive/_storage.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+            <label for="compute-infrastructure-Public cloud">Public cloud</label>
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/_storage.html]: Should be: compute-infrastructure-Public-cloud

File: templates/shared/forms/interactive/_storage.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+          <div class="col-3 u-sv3">
+            <input type="checkbox" id="storage-solutions-Other-NAS">
+            <label for="storage-solutions-Other-NAS">Other NAS</label>
+            <input type="checkbox" id="storage-solutions-Other-SAN">
+            <label for="storage-solutions-Other-SAN">Other SAN</label>
+            <input type="checkbox" id="storage-solutions-Local-storage">
+            <label for="storage-solutions-Local storage">Local-storage</label>
+            <div class="js-other-container">
+              <input type="checkbox" id="storage-solutions-other" class="js-other-container__checkbox">
+              <label for="storage-solutions-other">Other</label>              <input type="text" id="storage-solutions-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other">
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/_storage.html]: Can you put the input on a newline please

**Comment by pmahnke** [comment]
@anthonydillon what's the -1 for?

File: templates/shared/forms/interactive/_storage.html [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+          <div class="col-3 u-sv3">
+            <input type="checkbox" id="storage-solutions-Scality-RING">
+            <label for="storage-solutions-Scality-RING">Scality RING</label>
+            <input type="checkbox" id="storage-solutions-Caringo">
+            <label for="storage-solutions-Caringo">Caringo</label>
+            <input type="checkbox" id="storage-solutions-Portworx">
+            <label for="storage-solutions-Portworx">Portworx</label>
+            <input type="checkbox" id="storage-solutions-StorageOS">
+            <label for="storage-solutions-StorageOS">StorageOS</label>
+            <input type="checkbox" id="storage-solutions-Rook.io">
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/_storage.html]: Please replace `.` with `-`

**Pull request merged**